src/app/ui/shared/header/header.component.ts

```diff
@@ -6,10 +6,14 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./header.component.scss']
 })
 export class HeaderComponent implements OnInit {
+  headerTitle: string;
+  headerSubTitle: string;
 
   constructor() { }
 
   ngOnInit(): void {
+    this.headerTitle = 'Great products come by doing';
```

Comment:
Please move also this to constants - I think this is my code:D

Reply:
no longer used